packages/templates/wasm/typescript/shims/msgpack.js

```diff
@@ -0,0 +1,2356 @@
+module.exports = `function utf8Count(str) {
```

Comment:
There seem to be a lot of global functions being added here that are not apart of JS's globals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects

I worry that these will conflict with other functions used within user code.

I think a better way to solve this is to provide a "wrap lib" for typescript wrap devs to use, similar to how we do for Rust, AssemblyScript, and Go.